src/format/command.rs

```diff
@@ -10,8 +12,44 @@ pub fn run(stdin: bool, check: bool, files: Vec<String>) -> Result<(), Error> {
     }
 }
 
-pub fn format_files(_files: Vec<String>, _check: bool) -> Result<(), Error> {
-    todo!()
+pub fn format_files(files: Vec<String>, _check: bool) -> Result<(), Error> {
+    for file_path in files {
+        let path = PathBuf::from_str(&file_path).map_err(|e| Error::FileIO {
+            action: FileIOAction::Open,
+            kind: FileKind::File,
+            path: PathBuf::new(),
+            err: Some(format!("Invalid path: {}\n{}", file_path, e.to_string())),
+        })?;
+
+        let mut file = OpenOptions::new()
+            .write(true)
+            .read(true)
+            .open(path.clone())
+            .expect(&format!("Could not open file: {}", file_path));
```

Comment:
It would be good to `Error` for these error cases rather than crashing the program so the user has a better idea how to solve the problem when this happens.